source/extensions/clusters/redis/redis_cluster.cc

```diff
@@ -187,12 +187,16 @@ ProcessCluster(const NetworkFilters::Common::Redis::RespValue& value) {
 
   std::string address = array[0].asString();
   bool ipv6 = (address.find(':') != std::string::npos);
-  if (ipv6) {
-    return std::make_shared<Network::Address::Ipv6Instance>(address, array[1].asInteger());
+  try {
+    if (ipv6) {
+      return std::make_shared<Network::Address::Ipv6Instance>(address, array[1].asInteger());
```

Comment:
Sorry just realized, can you use `parseInternetAddress` here and avoid the ipv6 detection logic?

/wait-any

Reply:
Thanks for letting me know, I don't know why I didn't find this function I last looked.  Let me change that now.